elements/pl-graph/pl-graph.css

```diff
@@ -9,48 +9,3 @@
 .pl-graph > * {
   margin: auto;
 }
```

Comment:
When I removed this section, the rendered graph moved to the left side of the page. Can this margin line be added into the above class? (My CSS is not great).

Reply:
Adding the `mx-auto` class to the `div` should work, can you give that a shot?

Reply:
Can we get rid of the other styling options as well?

Reply:
Ok, let me run this locally and play around a bit.

Reply:
Try adding `align-items: center` to the `.pl-graph` styles? You can also change `overflow-x: scroll` to `overflow-x: auto` to avoid showing a scrollbar unless it's actually needed.

Reply:
ah, yeah, idk what's happening there... ok, let's leave the old styles, but maybe just update it to `.pl-graph > svg` to be specific?

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/7326/commits/32e116b39ee162594d09c12bbf9b9a69fbd6004a